_posts/2017-11-27-Babel-7-and-TypeScript.markdown

```diff
@@ -0,0 +1,261 @@
+---
+layout: post
+title: "Babel 7 + TypeScript"
+date: 2017-11-27 14:18
+comments: true
+author: chris
+github-url: https://www.github.com/damassi
+twitter-url: http://twitter.com/damassi
+categories: [Babel, TypeScript]
+---
+
+At Artsy we <3 [TypeScript](https://www.typescriptlang.org/). We use it with React Native via [Emission](https://github.com/artsy/emission) and on the web via [Reaction](https://github.com/artsy/reaction). Until recently, however, existing projects built on top of Babel had to implement convoluted tooling pipelines in order to work with the TypeScript compiler, increasing friction in an already complex landscape. Thankfully, those days [are over](https://github.com/babel/babel/tree/master/packages/babel-preset-typescript). Read on for an example project, as well as some advice on how to avoid common pitfalls when working with the new beta version of Babel 7.
```

Comment:
I think it makes sense to replace “projects built on top of Babel” with “projects that need Babel, because of [EXAMPLES]”, otherwise it may not be clear to the reader why we’d go through this hassle at all.

An example in e.g. Emission and Reaction is the Relay babel plugin.

Reply:
Will do